helloworld-mutual-ssl-secured/configure-client-certs.cli

```diff
@@ -0,0 +1,19 @@
+# Configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
```

Comment:
Configure the client's keystore. This will be used to generate the client's certificate. The path to the keystore file doesn’t actually have to exist yet

Reply:
Fixed!